lib/proxy/router.go

```diff
@@ -290,6 +290,7 @@ func (r *Router) DialHost(ctx context.Context, clientSrcAddr, clientDstAddr net.
 		if port == "" || port == "0" {
 			port = strconv.Itoa(defaults.SSHServerListenPort)
 		}
+		isAgentlessNode = true
```

Comment:
Can you add a comment here that states why this is added, and that this node is not actually an agentless node? Better yet rename `isAgentlessNode` to `isOpenSSHNode`, that makes more sense now

Reply:
We later assign it to the `isAgentlessNode` on params so I think it makes sense to keep the same name, and since it's a temporary change for the transition period (while we support TELEPORT_UNSTABLE_UNLISTED_AGENT_DIALING) I wouldn't want to rename it on params. Also, I think it technically is agentless, since there's no agent :) , but it's just non registered at teleport as well

Reply:
I've added a comment.

Reply:
Makes sense, thanks